docs/spl-docs/gentimes-command.md

```diff
@@ -0,0 +1,153 @@
+# gentimes command
+
+## Description
+Generates timestamp results beginning at the specified start time. Each result represents an adjacent, non-overlapping time range based on the increment value. This process continues until enough results are generated to pass the end time.
+
+The `gentimes` command produces events up to, but not including, the end time.
+
+## Syntax
+
+Required syntax is in **bold**.
+
+| **gentimes**\
+**start=\<timestamp\>**\
+[end=\<timestamp\>]\
+[increment=\<increment\>]
+
+### Required Arguments
+
+#### start
+**Syntax:** `start=<timestamp>`\
+**Description:** \
+Specify the start time.
+
+#### \<timestamp\>
+**Syntax:** `MM/DD/YYYY[:HH:MM:SS]` or `<int>`\
+**Description:**\
+Specify the timeframe using either a timestamp or an integer value.\
+For example: `11/15/2021` for November 15, 2021 at 00:00:00, `05/20/2022:14:30:00` for May 20, 2022 at 14:30:00, or `-7` for seven days ago.
```

Comment:
> Specify the timeframe using either a timestamp or an integer value.

nit: I think it would be good to specify what the integer value means here. Because from just this sentence I would guess it's the unix epoch, but from line 28 it seems it's actually in units of days relative to now

Reply:
addressed in the latest commit.